app/src/main/java/com/framgia/englishconversation/screen/basePost/BasePostViewModel.java

```diff
@@ -326,6 +326,10 @@ public void onSubmitPost() {
         updateTimelineModel();
         if (mAdapterType == AdapterType.CONVERSATION) {
             mTimelineModel.setConversations(mConversationAdapter.getValidatedData());
+            mTimelineModel.setMedias(null);
+        }
```

Comment:
ERROR: '}' at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).